### PR TITLE
Rename help string `parachain-collator` -> `parachain-template-node`

### PR DESCRIPTION
**Pull request by skunert**
Just some leftovers from #1303

### CODE REVIEW

**Review by KiChjang — APPROVED** [review]

File: parachain-template/node/src/command.rs [hunk 1]
```diff
@@ -45,7 +45,7 @@ impl SubstrateCli for Cli {
 		"Parachain Collator Template\n\nThe command-line arguments provided first will be \
 		passed to the parachain node, while the arguments provided after -- will be passed \
 		to the relay chain node.\n\n\
-		parachain-collator <parachain-args> -- <relay-chain-args>"
+		parachain-template-node <parachain-args> -- <relay-chain-args>"
```
Comment (davxy) [inline parachain-template/node/src/command.rs:48]: Nit (feel free to ignore). Maybe you can use the same technique that has been used here

https://github.com/paritytech/cumulus/blob/17ca448df53c76556ff85538f59f850c3fdd6f01/polkadot-parachain/src/command.rs#L179-L186

**Review by davxy — APPROVED** [review]

**Review by koute — APPROVED** [review]

**Comment by shawntabrizi** [comment]
`parachain-node-template` matches better with `substrate-node-template`?

File: parachain-template/node/src/command.rs (hunk 1 above)
Comment (skunert) [reply parachain-template/node/src/command.rs:48]: Will do

**Comment by skunert** [comment]
@shawntabrizi I do not see much discussion in the original issue https://github.com/paritytech/cumulus/pull/1303, so not sure if this was considered. I agree it would be nicer to have this aligned. Worth renaming again?

**Review by gilescope — APPROVED** [review]
LGTM

**Pull request merged**